### PR TITLE
Add face for mini-modeline and perspective

### PR DESCRIPTION
**Pull request by kaiwk**
Add support for [mini-modeline](https://github.com/kiennq/emacs-mini-modeline) and [perspective-el](https://github.com/nex3/perspective-el).

### CODE REVIEW

File: color-theme-sanityinc-tomorrow.el [hunk 1]
```diff
@@ -1189,6 +1192,9 @@ names to which it refers are bound."
       ;; macrostep
       (macrostep-expansion-highlight-face (:inherit highlight :foreground nil))
 
+      ;; mini-modeline
+      (mini-modeline-mode-line (:background ,foreground :height 0.1 :box nil))
```
Comment (purcell) [inline color-theme-sanityinc-tomorrow.el:1196]: Shouldn't this `:inherit mode-line`?

File: color-theme-sanityinc-tomorrow.el [hunk 2]
```diff
@@ -575,6 +575,9 @@ names to which it refers are bound."
       (show-paren-match (:background ,purple :foreground ,background))
       (show-paren-mismatch (:background ,red :foreground ,background))
 
+      ;; perspective
+      (persp-selected-face (:weight bold :foreground ,yellow))
```
Comment (purcell) [inline color-theme-sanityinc-tomorrow.el:579]: These new sections should be added in positions that preserve alphabetical order, please.

**Review by purcell — COMMENTED** [review]
Thanks for this - left a couple of comments.

**Comment by kaiwk** [comment]
Fixed the issues in the comments above.

**Comment by purcell** [comment]
Super, thanks!

**Pull request merged**